index.js

```diff
@@ -34,15 +33,16 @@ github.authenticate({
     token: githubApiToken
 });
 
-const createComment = (number, assignee) => {
+const createComment = (number, message, assignee) => {
   if (dryRun) {
-    console.log(`DRYRUN: would comment on ${number} for ${assignee}`);
+    console.log(`DRYRUN: would comment on ${number} for ${assignee}:
+${assignmentMessage(message, assignee)}`);
```

Comment:
Does backtick syntax have "this new line starts here" support? If it's like other languages it would be a pipe symbol. If so can you align the second line with the start of the first? Or alternatively, just put a newline at the end of the first message and then have the assignment message as part of the second thing to console.log, i.e.
```js
console.log(`DRYRUN: would comment on ${number} for ${assignee}:\n`, assignmentMessage(message, assignee));
```